cyclops/models/wrappers/sk_model.py

```diff
@@ -47,10 +56,22 @@ class SKModel:
 
     """
 
-    def __init__(self, model: SKBaseEstimator, **kwargs) -> None:
+    def __init__(  # pylint: disable=dangerous-default-value
+        self,
+        model: SKBaseEstimator,
+        model_params: dict,
+        fit_params: dict = {},
+        best_model_params: dict = {},
```

Comment:
Why not pass `best_model_params` and `fit_params` during hyperparameter tuning and model fitting, respectively?

Reply:
Because they were already in the config file. Also I wanted to remove batch_size from the methods parameters so I thought similar to PTModel, add have everything set up from the beginning. You think it's not a good idea?

Reply:
Yeah. I think the difference in initialization style is consistent with the workflow in the two frameworks.